src/styles/rating-indicator.scss

```diff
@@ -25,46 +20,40 @@ $block: #{$fd-namespace}-rating-indicator;
   $fd-rating-indicator-dynamic-text-space: 0.5rem;
 
   @mixin fd-rating-indicator-with-half-star($iconFontSize, $spacingBetweenIcons) {
-    .#{$block}__input {
-      width: $iconFontSize / 2;
-      height: $iconFontSize;
-    }
+    .#{$block} {
+      &__input {
+        width: $iconFontSize * 0.5;
+        height: $iconFontSize;
+      }
 
-    .#{$block}__label {
-      margin-right: 0;
+      &__label {
+        margin-right: 0;
```

Comment:
no rtl?

Reply:
you can use a mixin for setting the margins, they will handle the RTL. 
You can check `fd-set-position-right` and `fd-set-position-right-pseudo`